configure

```diff
@@ -579,10 +580,6 @@ def configure_arm(o):
     o['variables']['arm_fpu'] = 'vfpv3'
     o['variables']['arm_version'] = '7'
 
-  # Print warning when snapshot is enabled and building on armv6
-  if is_arch_armv6() and options.with_snapshot:
-    warn('when building on ARMv6, don\'t use --with-snapshot')
-
```

Comment:
The default is still to build without snapshot, right? I'm sorry, but the diff doesn't make it obvious for me.